lib/fakeroot.py

```diff
@@ -224,22 +245,46 @@
      "cmds": ["apt", "apt-get", "dpkg"],
      "each": ["fakeroot"] },
 
-   # Fedora notes:
-   #
-   # 1. The minimum supported version was chosen somewhat arbitrarily based on the
-   #    release versions available for testing (what was on Docker Hub).
-   #
-   # 2. The fakeroot package is in the base repository set so enabling EPEL is
-   #    not required.
-
-   "fedora":
-   { "name": "Fedora 24+,",
-     "match":  ("/etc/fedora-release", r"release (?!1?[0-9] |2[0-3] )"),
+   "suse":
+   { "name": "(Open)SUSE 42.2+",  # no fakeroot before this
+     # I don't know if there are OpenSUSE derivatives
+     "match": ("/etc/os-release", r"ID_LIKE=.*suse"),
      "init": [ ("command -v fakeroot > /dev/null",
-                "dnf install -y fakeroot") ],
-     "cmds": ["dnf", "rpm", "yum"],
+                # fakeroot seems to have a missing dependency, otherwise
+                # failing with missing getopt in the fakeroot script.
+                "zypper refresh; zypper install -y fakeroot /usr/bin/getopt") ],
+     "cmds": ["zypper", "rpm"],
      "each": ["fakeroot"] },
 
+   # no worky; see #1295
+   # [It seems to me (Dave Love) that it works sufficiently well, just
+   # failing to set filesystem ownership/permissions that presumably
+   # aren't necessary for what Charliecloud addresses.  Perhaps the spec
+   # could be extended to allow showing a message about such things.
+   # However, at least mostly, --force seems not to be necessary.]
+
+   # pacman doesn't seem to have proper dependencies like dpkg and
+   # rpm.  It happens that fakeroot can fail because the downloaded
+   # version is linked against a newer glibc (at least) than is in the
+   # base image. Thus update that too, and for safety also the
+   # util-linux dependency.  We could add -u to update all installed
+   # packages, but that may not be what a user wants.
+   # "arch":
+   # { "name": "Arch Linux",
+   #  "match": ("/etc/os-release", r"ID=arch"),  # /etc/arch-release empty
+   #  "init": [ ("command -v fakeroot > /dev/null",
+   #             "pacman -Syq --noconfirm glibc util-linux fakeroot") ],
+   #  "cmds": ["pacman"],
+   #  "each": ["fakeroot"] },
+
+   # no worky; see #1296
+   #"alpine":
+   #{ "name": "Alpine, any version",
```

Comment:
```suggestion
   # "alpine":
   # { "name": "Alpine, any version",
```

Be consistent with previous comment spacing.

Reply:
I actually prefer the no-space version for commenting out code, because it provides a visual break from possibly preceding text comments.